lib/h3/traversal.rb

```diff
@@ -262,6 +279,30 @@ def k_ring_distances(origin, k)
       ]
     end
 
+    # Derives the H3 indexes found in the line between an origin H3 index
```

Comment:
Instead of "the line", I believe it would be more accurate to say "a line", since the docs of H3 make clear there is no one single (minimal) line in general between two hexagons. They acknowledge that when they say `h3Line` and friends are guaranteed to have the same length between lib upgrades, but the actual hexagons may be different. They only guarantee a line is minimal and is connected (you only move from neighbor to neighbor).

Reply:
In a plane or sphere, there is only one segment or maximal arc going from `origin` to `destination`, but in the hexagonal tesselation you move doing zig-zag and there may be more that one "line" with the same number of minimal hexagons. You lose unicity.

Reply:
Thanks - that makes it easier to understand 👍 I'll update the doc accordingly.